.github/workflows/release.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       - name: "Set up Python"
         uses: actions/setup-python@v2
       - name: "Install Python modules"
-        run: pip3 install PyGithub requests python-dateutil
+        run: pip3 install PyGithub python-dateutil
```

Comment:
`PyGithub` requires `requests`, so we don't need to separately install it. And anyway, we don't even use the `requests` module directly in any of our release scripts, except for `update-website.py`, but that script isn't run in this workflow. So it's simpler to just remove it.